library/src/scala/tasty/Reflection.scala

```diff
@@ -452,7 +452,8 @@ class Reflection(private[scala] val internal: CompilerInterface) { self =>
   //////////////
 
   /** Context of the macro expansion */
-  given rootContext as Context = internal.rootContext // TODO: Use given // TODO: Should this be moved to QuoteContext?
+  def rootContext: Context = internal.rootContext // TODO: Should this be moved to QuoteContext?
+  given Context = rootContext // TODO: Should be an implicit converion from QuoteContext to Context
```

Comment:
This change seems not useful.

Reply:
It is to be able to access the `rootContext` by name when only importing `import qctx.tasty._` , otherwise it is only visible with the given import.

Reply:
Nothing changed for summoning it